assets/base.css

```diff
@@ -3279,12 +3280,12 @@ details-disclosure > details {
 
   .scroll-trigger.animate--fade-in,
   .scroll-trigger.animate--slide-in {
-    transition: opacity var(--duration-extra-long) ease-out;
-    opacity: 0;
+    opacity: 0.1;
```

Comment:
I'm seeing some stuttering on the demo link, specifically on `slide-in` animations.

Do you think this is related to this PR?

https://user-images.githubusercontent.com/37168033/234676244-3f64f652-df77-4645-9c29-1c634d387b23.mp4

Reply:
Good catch 👍 pushed the fix.